src/_locales/en/messages.json

```diff
@@ -138,10 +138,10 @@
     "message": "Find shortcuts for this extension by going to <a>extension settings.</a>"
   },
   "welcomePageTryFirefox": {
```

Comment:
Since this string is changing and will require an updated translation, we will need to update the string ID to trigger retranslations in Pontoon.